modules/web/src/app/shared/entity/cluster.ts

```diff
@@ -56,6 +56,11 @@ const PROVIDER_DISPLAY_NAMES = new Map<Provider, string>([
   [Provider.VMwareCloudDirector, 'VMware Cloud Director'],
 ]);
 
+// Internal annotation used by KKP for system-level configuration.
+export const InternalclusterSpecAnnotations = {
```

Comment:
Please also fix any other variable names across other files.
```suggestion
export const InternalClusterSpecAnnotations = {
```